packages/wallet/src/KeyManagement/types.ts

```diff
@@ -72,7 +89,7 @@ export type GetPassword = (noCache?: true) => Promise<Uint8Array>;
 export interface KeyAgent {
   get networkId(): Cardano.NetworkId;
   get accountIndex(): number;
-  get serializableData(): SerializableKeyAgentData;
+  get serializableData(): SerializableKeyAgentData | SerializableLedgerKeyAgentData;
```

Comment:
It would be better to do: `export type SerializableKeyAgentData = SerializableInMemoryKeyAgentData | SerializableLedgerKeyAgentData;`